content/terms/explanation/filters.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Filters"
+weight: 3
+---
+
+# Filters
+
+Filters solve noise issues in terms versions that cannot be addressed with direct selection or removal of content using CSS selectors or range selectors.
+
+## Why filters are needed
+
+Web pages often contain dynamically generated content or content that cannot be targeted with CSS selectors that creates noise in the archive:
+
+- Tracking parameters in URLs, for example `utm_source`, `utm_medium`, …
+- Content that are date based and can change between visits, for example "Updated X days ago" can be converted to a "Last updated on YYYY-MM-DD".
+- Dynamic elements with changing classes or IDs
+
+Without filters, this dynamic content creates changes that are not meaningful to the terms.
+
+## How filters work
+
+Filters are JavaScript functions that receive a JSDOM document instance and can manipulate the DOM structure directly. They modify the document structure and content in-place and they run sequentially in the order specified in the declaration.
+
+## Filter design principles
+
+When designing filters, follow these core principles:
+
+- **Be specific**: Target only the noise you want to remove. Avoid broad selectors that might accidentally remove important content.
+- **Be safe**: Ensure your filter doesn't accidentally remove important content. Always check that the generated version still contains the whole terms content.
+- **Be idempotent**: Your filter should produce the same result even if run multiple times on its own output. This ensures consistency and prevents unexpected behavior.
+- **Be efficient**: Use efficient DOM queries and avoid unnecessary operations. Process only the elements you need to modify.
+
+## When to use filters
```

Comment:
_Why_ and _When_ sections seems to be a little bit similar. I wonder if it could be grouped together.